### ISSUE TITLE
Source columns have unreadable landmark names

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7805]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.822
    OS Version      : any OS running RStudio Server
    R Version       : any

### Steps to reproduce the problem
- start screen reader (I'm using VoiceOver on Safari)
- load RStudio Server
- add a couple source columns via "Add Source Column" command
- in screen reader, bring up list of Landmark regions (on VoiceOver, use VO+U and arrow keys)

### Describe the problem in detail
- the source columns have names like "Source3ir06540t415 region" and "Sourcebaz9otp6zz75 region"

![screenshot of VoiceOver landmarks rotor showing bogus source column names](https://user-images.githubusercontent.com/10569626/93278086-86ec7100-f778-11ea-8a16-ddf2b8676f70.jpeg)

### Describe the behavior you expected
- regions should have distinct but meaningful names

### Other symptoms
- The min/max buttons in the panel header also use this label, so if you tab to them you get "Minimize Source3ir06540t415", etc.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by rich-rstudio** [comment]
verified on server 1.4.890
